modules/data-storage-management.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * storage/optimizing-storage.adoc
+
+[id="data-storage-management_{context}"]
+= Data storage management
+
+The following table summarizes the main directories that {product-title} components write data to.
+
+.Main directories for storing {product-title} data
+[options="header,footer"]
+|===
+|Directory|Notes|Sizing|Expected growth
+
+|*_/var/lib/openshift_*
+|Used for etcd storage only when in single master mode and etcd is embedded in the *atomic-openshift-master* process.
+|Less than 10 GB.
+|Will grow slowly with the environment. Only storing metadata.
+
+|*_/var/lib/etcd_*
+|Used for etcd storage when in Multi-Master mode or when etcd is made standalone by an administrator.
+|Less than 20 GB.
+|Will grow slowly with the environment. Only storing metadata.
+
+Additional 20-25 GB for every additional 8 GB of memory.
+|Growth is limited by the capacity for running containers.
+
+|*_/var/lib/containers_*
+|When the runtime is CRI-O, this is the mount point. Storage used for active container runtimes (including Pods) and storage of local images (not used for registry storage).
```

Comment:
CRI-O is IMO the only runtime we support, there is no "When".

Reply:
Changed wording to "This is the mount point for the CRI-O runtime."